Makefile.env.mk

```diff
@@ -39,7 +39,19 @@ PROMETHEUS_IMAGE ?= "prom/prometheus:v2.4.3"
 ALERTMANAGER_IMAGE ?= "prom/alertmanager:v0.15.2"
 GRAFANA_IMAGE ?= "grafana/grafana:5.3.1"
 KUBE_STATE_METRICS_IMAGE ?= "quay.io/coreos/kube-state-metrics:v1.4.0"
+QDROUTERD_BASE_IMAGE ?= "enmasseproject/qdrouterd-base:1.5-rc2"
```

Comment:
This is already out of date, assuming you want to use the same as the router base :)